### PR TITLE
Skew t hydro vert sums

### PR DESCRIPTION
**Pull request by cshartsough**
Added vertically integrated mixing ratios table to the skewT hydrometeor subplot.  Also changed the hydrometeor legend to match the symbols in the figure.

Tests completed without error.  See sample below.

Screen Shot 2021-03-16 at 9 37 05 AM

### CODE REVIEW

**Review by christinaholtNOAA — COMMENTED** [review]
Looks good!

**Comment by Brian-Jamison** [comment]
Great work!

**Comment by christinaholtNOAA** [comment]
Hey @cshartsough, what's the status here? Should we discuss in the meeting today, or is this ready?

**Comment by cshartsough** [comment]
I still need to redo the vertical integration step and include the model name in the title.  I should have time to work on it this week, after I catchup on some other tasks that had slowed things down.

**Comment by cshartsough** [comment]
new updated plot
Screen Shot 2021-03-31 at 5 18 51 PM

File: create_graphics.py [hunk 1]
```diff
@@ -203,6 +203,13 @@ def parse_args():
         required=True,
         type=int,
         )
+    parser.add_argument(
+        '-m',
+        default='hrrr',
+        dest='model_name',
+        help='model_name used to plotting.',
```
Comment (christinaholtNOAA) [inline create_graphics.py:210]: Maybe "string to use in title of graphic"??

**Review by christinaholtNOAA — APPROVED** [review]
This looks great! Had to get real picky to find anything. :)

File: create_graphics.py (hunk 1 above)
Comment (cshartsough) [reply create_graphics.py]: thanks, made the change.

**Review by Brian-Jamison — APPROVED** [review]
Nice work!  Looks fantastic!

**Pull request merged**